lib/api/core/storage/bootstrap/internalIndexBootstrap.js

```diff
@@ -108,6 +110,9 @@ class InternalIndexBootstrap extends SafeBootstrap {
 
     await this._persistJWTSecret();
 
+    // @todo test unit
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)